app/scripts/views/mixins/checkbox-mixin.js

```diff
@@ -17,6 +18,12 @@ define([
       'change input[type=checkbox]': 'logCheckboxChange'
     },
 
+    afterRender: function () {
+      this.$el.find('.fxa-checkbox').get().forEach(function (el) {
```

Comment:
Retrieves the DOM nodes as an array, see https://api.jquery.com/get/#get2 for more info